### ISSUE TITLE
Allow changing pause modal text

### ISSUE THREAD

**Issue by LaurensBurger** [open-formulieren/open-forms#4515]
### Thema / Theme

Frontend

### Omschrijving / Description

internal: Hogeland 20

For municipalities who do not use "u"/"uw" in their communication the text in the pause modal should be configurable. 
![image](https://github.com/open-formulieren/open-forms/assets/112621537/25c3933d-2334-4ca4-96bb-d02750e6f52f)

**Comment by sergei-maertens** [comment]
Can we not just change the text to the informal variant instead? We've been gradually moving in that direction anyway.

**Comment by joeribekker** [comment]
What sergei said. Not making this configurable.
